.github/workflows/e2e-android-app.yml

```
@@ -0,0 +1,96 @@
+name: Functional Android App Tests
+
+on: [pull_request]
+
```

How about also having them run on `main`?

```suggestion
on:
  push:
    branches:
      - main
  pull_request:
```

---

My PR is here as a demonstration to help mainainer of this repo. It is welcome to redo the PR to fit with the contribution guidelines. I have not tested this but it should work

---

> Any reason why this workflow is not triggered for this PR?

Because it's not in the same repo.

I merged your suggestion